docs/_source/cover.rst

```diff
@@ -7,66 +7,67 @@ Xpanse Python SDK
 Overview
 --------
 
-This library is intended to be an interface to the `Xpanse Expander API <https://knowledgebase.expanse.co/expander-apis/>`_.
```

Comment:
again, Cortex Xpanse vs Xpanse Expander.

Reply:
great finds, thank you!